src/apis/googleMapsApi.ts

```diff
@@ -0,0 +1,12 @@
+
+import apiConsts from "../consts/apiConsts";
+export const  googleMapsApi = (location: string) =>{
+    const API_KEY_GOOGLE = apiConsts.GOOGLE_KEY;
+    const API_GOOGLE = apiConsts.API_URL_GOOGLE_GEOCODING;
```

Comment:
You can use .env to keep some api keys then call them with `process.env.XXXXXX`

Reply:
ok i'll modify it